cpu/stm32l1/vectors.c

```diff
@@ -29,130 +28,124 @@ void dummy_handler(void) {
+    [DMA1_Channel4_IRQn      ] = isr_dma1_channel4,        /* [14] DMA1 Channel 4 global Interrupt */
+    [DMA1_Channel5_IRQn      ] = isr_dma1_channel5,        /* [15] DMA1 Channel 5 global Interrupt */
+    [DMA1_Channel6_IRQn      ] = isr_dma1_channel6,        /* [16] DMA1 Channel 6 global Interrupt */
+    [DMA1_Channel7_IRQn      ] = isr_dma1_channel7,        /* [17] DMA1 Channel 7 global Interrupt */
+    [ADC1_IRQn               ] = isr_adc1,                 /* [18] ADC1 global Interrupt */
+    [USB_HP_IRQn             ] = isr_usb_hp,               /* [19] USB High Priority Interrupt */
+    [USB_LP_IRQn             ] = isr_usb_lp,               /* [20] USB Low Priority Interrupt */
+    [DAC_IRQn                ] = isr_dac,                  /* [21] DAC Interrupt */
+    [COMP_IRQn               ] = isr_comp,                 /* [22] Comparator through EXTI Line Interrupt */
+    [EXTI9_5_IRQn            ] = isr_exti,                 /* [23] External Line[9:5] Interrupts */
+    [TIM9_IRQn               ] = isr_tim9,                 /* [25] TIM9 global Interrupt */
+    [TIM10_IRQn              ] = isr_tim10,                /* [26] TIM10 global Interrupt */
+    [TIM11_IRQn              ] = isr_tim11,                /* [27] TIM11 global Interrupt */
+    [TIM2_IRQn               ] = isr_tim2,                 /* [28] TIM2 global Interrupt */
+    [TIM3_IRQn               ] = isr_tim3,                 /* [29] TIM3 global Interrupt */
+    [TIM4_IRQn               ] = isr_tim4,                 /* [30] TIM4 global Interrupt */
+    [I2C1_EV_IRQn            ] = isr_i2c1_ev,              /* [31] I2C1 Event Interrupt */
+    [I2C1_ER_IRQn            ] = isr_i2c1_er,              /* [32] I2C1 Error Interrupt */
+    [I2C2_EV_IRQn            ] = isr_i2c2_ev,              /* [33] I2C2 Event Interrupt */
+    [I2C2_ER_IRQn            ] = isr_i2c2_er,              /* [34] I2C2 Error Interrupt */
+    [SPI1_IRQn               ] = isr_spi1,                 /* [35] SPI1 global Interrupt */
+    [SPI2_IRQn               ] = isr_spi2,                 /* [36] SPI2 global Interrupt */
+    [USART1_IRQn             ] = isr_usart1,               /* [37] USART1 global Interrupt */
+    [USART2_IRQn             ] = isr_usart2,               /* [38] USART2 global Interrupt */
+    [USART3_IRQn             ] = isr_usart3,               /* [39] USART3 global Interrupt */
+    [EXTI15_10_IRQn          ] = isr_exti,                 /* [40] External Line[15:10] Interrupts */
+    [RTC_Alarm_IRQn          ] = isr_rtc_alarm,            /* [41] RTC Alarm through EXTI Line Interrupt */
+    [USB_FS_WKUP_IRQn        ] = isr_usb_fs_wkup,          /* [42] USB FS WakeUp from suspend through EXTI Line Interrupt */
+    [TIM6_IRQn               ] = isr_tim6,                 /* [43] TIM6 global Interrupt */
+    [TIM7_IRQn               ] = isr_tim7,                 /* [44] TIM7 global Interrupt */
+
```

Comment:
Any special reason fo the extra space here and below? (just curiosity, I'm not against it)